validator/src/journal/block_manager.rs

```diff
@@ -430,6 +430,14 @@ impl BlockManager {
         Box::new(GetBlockIterator::new(Arc::clone(&self.state), block_ids))
     }
 
+    pub fn get_block_from_blockstore(
```

Comment:
block seems redundent; `get_from_blockstore` would be fine

Reply:
Sounds good.